src/app.tsx

```diff
@@ -23,22 +25,24 @@ export const App: React.FC = () => {
   return (
     <ThemeProvider theme={theme}>
       <GlobalStyles />
-      <ColorModeProvider defaultMode="light">
-        <>
-          <Router>
-            <AppErrorBoundary>
-              <VaultLoader />
-              <Routes />
-              <AccountsDrawer />
-              <NetworksDrawer />
-              <TransactionSettingsDrawer />
-              <SpeedUpTransactionDrawer />
-              <SettingsPopover />
-            </AppErrorBoundary>
-            <Toaster position="bottom-center" toastOptions={{ style: { fontSize: '14px' } }} />
-          </Router>
-        </>
-      </ColorModeProvider>
+      <QueryClientProvider client={queryClient}>
```

Comment:
just for compat, we should use the query client exported from `jotai-query-toolkit`

Reply:
Should we be continuing to implement new uses of the toolkit if we are moving away from atoms with queries? Won't the use of that toolkit be phased out?

Reply:
I think we can use the one from jotai-query-toolkit for now to keep compatibility and same caching. We can easily remove it later.